group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorShardTest.java

```diff
@@ -293,7 +326,11 @@ public void testReplayOffsetCommit() {
             new ApiMessageAndVersion(value, (short) 0)
         ));
 
-        verify(offsetMetadataManager, times(2)).replay(key, value);
+        verify(offsetMetadataManager, times(2)).replay(
+            RecordBatch.NO_PRODUCER_ID,
```

Comment:
should we verify something with a producer ID here?

Reply:
Sounds good. Added testReplayTransactionalOffsetCommit.